ToDoListTake2/Program.cs

```diff
@@ -170,7 +169,7 @@ static void Main(string[] args)
                             Console.WriteLine($"id: {item.id} | task: {item.task} | status: {item.status} | scheduled_for: {item.scheduled_for}");
                         }
                         Console.WriteLine("What task id number would you like to update in your ToDo List? (Hint: enter the number of the id.");
-                        while(!int.TryParse(Console.ReadLine(), out taskId))
+                        while (!int.TryParse(Console.ReadLine(), out taskId))
```

Comment:
Sorry I can't comment directly on line 180, but I wonder if it would be more helpful to ask the user for a date they plan to accomplish the task or do envision this as only a weekly ToDo List? If so maybe the wording could be "day of the week"

Reply:
I did imagine day of the week.